googlesource/googlesource.go

```diff
@@ -38,44 +37,40 @@ var (
 type readRowInput struct {
 	tableID   string
 	offset    string
-	positions map[string]string
+	positions string
 	wg        *sync.WaitGroup
 }
 
 // checkInitialPos helps in creating the query to fetch data from endpoint
-func (s *Source) checkInitialPos(positions map[string]string, incrementColName map[string]string, tableID string, primaryColName map[string]string) (firstSync, userDefinedOffset bool, userDefinedKey bool) {
+func (s *Source) checkInitialPos(positions string, incrementColName string, tableID string, primaryColName string) (firstSync, userDefinedOffset bool, userDefinedKey bool) {
```

Comment:
TableID not used anymore. Also, the function is using named return values, so we can use a naked return here. Or we don't use named return values.